packages/create-skill-app/templates/next/src/trpc/routers/abilities.ts

```diff
@@ -0,0 +1,30 @@
+import {getToken} from 'next-auth/jwt'
+import {
+  defineRulesForPurchases,
+  UserSchema,
+} from '@skillrecordings/skill-lesson/utils/ability'
+
+import {getSubscriberFromCookie} from '@skillrecordings/skill-lesson/utils/ck-subscriber-from-cookie'
+import {convertkitSetSubscriberCookie} from '@skillrecordings/skill-lesson/utils/ck-set-subscriber-cookie'
+import {publicProcedure, router} from '@skillrecordings/skill-lesson'
+
+export const abilities = router({
+  getAbilities: publicProcedure.query(async ({ctx, input}) => {
+    const token = await getToken({req: ctx.req})
+    const convertkitSubscriber = await getSubscriberFromCookie(ctx.req)
+
+    // TODO: where could we move this? It doesn't seem like an ability check is
+    // the best place to set a cookie.
+    convertkitSetSubscriberCookie({
+      subscriber: convertkitSubscriber,
+      res: ctx.res,
+    })
```

Comment:
it's "just a request" and it's more of a "refreshing the cookie with a new expiration" I think, so seems fine?

I actually think this might be totally useless here since the "get from cookie" isn't loading from CK or anything, which it might be nice if it did to refresh any fields etc.